.travis.yml

```diff
@@ -32,19 +32,21 @@ install:
   - pipenv install --dev
 
 # perform testing:
-# --> run the test suite with pytest
+# --> run the test suite with pytest and codecov
+# --> Check for black format
 # --> lint the functions with flake8
 # --> lint the test suites with flake8
 # --> lint the functions with pylint
 # --> lint the test suites with pylint
 # --> lint the README documentation with mdl
+# --> Run django server
 script:
   - pipenv run pytest gatorgrouper/tests --cov-config pytest.cov --cov
-  - pipenv run black */**.py --check
-  - pipenv run flake8 tests*
-  - pipenv run flake8 */**.py
-  - pipenv run pylint tests
-  - pipenv run pylint */**.py
+  - pipenv run black */*/*.py --check
```

Comment:
This path only checks files that are exactly 2 levels nested. It should be `**/*.py *.py`, which will check both files directly in the root (`*.py`), and files nested any level deep (`**/*.py`).

Reply:
Here is what it looks like for me
````
mohammad@mohammad-XPS-15-9550:~/Documents/cs203/develop/gatorgrouper$ echo **/*.py *.py
gatorgrouper/admin.py gatorgrouper/apps.py gator_grouper/__init__.py gatorgrouper/models.py gator_grouper/secret_key.py gator_grouper/settings.py gatorgrouper/tests.py gator_grouper/urls.py gatorgrouper/urls.py gatorgrouper/views.py gator_grouper/wsgi.py manage.py
mohammad@mohammad-XPS-15-9550:~/Documents/cs203/develop/gatorgrouper$ echo */*/*.py
gatorgrouper/tests/conftest.py gatorgrouper/tests/__init__.py gatorgrouper/tests/test_absent.py gatorgrouper/tests/test_arguments.py gatorgrouper/tests/test_create_escaped_string.py gatorgrouper/tests/test_display.py gatorgrouper/tests/test_group_method.py gatorgrouper/tests/test_group_size.py gatorgrouper/utils/defaults.py gatorgrouper/utils/display.py gatorgrouper/utils/gatorgrouper.py gatorgrouper/utils/group_random.py gatorgrouper/utils/group_rrobin.py gatorgrouper/utils/group_scoring.py gatorgrouper/utils/__init__.py gatorgrouper/utils/parse_arguments.py gatorgrouper/utils/read_student_file.py gatorgrouper/utils/rebalance.py gatorgrouper/utils/remove_absent_students.py
````

Reply:
Do you have `shopt -s globstar` enabled on your local computer?

Reply:
It seems likely after reading a few SO, we likely will want to use `find`.

(https://unix.stackexchange.com/questions/49913/recursive-glob)
(https://stackoverflow.com/questions/1690809/what-expands-to-all-files-in-current-directory-recursively)